util/src/main/java/tech/pegasys/artemis/util/cli/CommandLineArguments.java

```diff
@@ -73,18 +55,6 @@
         }
       };
 
```

Comment:
Move these Events to the toml file under output as well.

Reply:
Now i have moved events to config under output. pls review.